src/index.js

```diff
@@ -47,7 +133,6 @@ function clean(ast, newObj) {
 const printers = {
   python: {
     print,
-    hasPrettierIgnore: util.hasIgnoreComment,
```

Comment:
Note that this wasn't actually being passed in, as far as I can tell (the value was `undefined`, even on master).